analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -148,7 +148,7 @@ private ExplodedNode ProcessBranch(ExplodedNode node, ControlFlowBranch branch)
 
         private ProgramState ProcessBranchState(ControlFlowBranch branch, ProgramState state)
         {
-            foreach (var local in branch.EnteringRegions.SelectMany(x => x.Locals))
+            foreach (var local in branch.EnteringRegions.SelectMany(x => x.Locals).Where(x => x.Language == LanguageNames.VisualBasic)) // This is needed as we don't support tuple deconstructions yet
```

Comment:
Is it necessary to iterate all the way through if we are not VB? Can a region span multiple languages? If not, we can bail out after the first local.